packages/server/lib/gui/events.js

```diff
@@ -75,6 +79,20 @@ const handleEvent = function (options, bus, event, id, type, arg) {
   }
 
   switch (type) {
+    case 'get:package-manager': {
+      return getPackageManager()
+      .then((pkg) => {
+        return sendResponse({ type: 'success', event: 'get:package-manager', data: pkg })
```

Comment:
I think it would be good to have a standard way to communicate between server <-> desktop GUI. The standard payload might be:

```js
{
  type: 'success' | 'failure',
  data: any
  event: string
}
```

This makes it easy to have a type safe event emitter - see below.